### PR TITLE
Header footer

### PR DESCRIPTION
**Pull request by tori-calkins**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -80,3 +80,132 @@ h1{
...
+  &:hover{
+    border-top-right-radius:0px;
+    border-bottom-left-radius:0px;
+    color:black;
+    &:before,&:after{
+    width:100%;
+    height:100%;
+    border-color:black;
+      }
+    }
```
Comment (SunkissedQueen) [inline src/App.css]: indentation

File: src/App.css [hunk 2]
```diff
@@ -80,3 +80,132 @@ h1{
...
+  }
+  &:hover{
+    border-top-right-radius:0px;
+    border-bottom-left-radius:0px;
+    color:black;
+    &:before,&:after{
+    width:100%;
+    height:100%;
+    border-color:black;
+    }
```
Comment (SunkissedQueen) [inline src/App.css]: indentation

File: src/component/Header1.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                    </Dropdown>
+                </div>
+            </Navbar>
+        </>
+    )
+}
+
+Header.propTypes = {
+    direction: PropTypes.string,
+  };
```
Comment (SunkissedQueen) [inline src/component/Header1.js]: this is interesting...what purpose does it serve?

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Oh my gosh!!! I can't wait to see this website. 
Just need some tweaks: indentation.
I have a concern about proptypes placement. Since it does not need to be in the return statement. I think it should be located above the return...for a cleaner look. Best practices is to place custom variables, state variables, functions between the functional component declaration and the return statement.
```js
Header2.propTypes = {
    direction: PropTypes.string,
  };
```

File: src/component/Header1.js (hunk 3 above)
Comment (tori-calkins) [reply src/component/Header1.js]: it was a reactstrap attribute they had for the dropdown.

**Review by SunkissedQueen — APPROVED** [review]

**Pull request merged**